Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -201,6 +386,59 @@ - (void)verifyClientWithCompletion:(FIRVerifyClientCallback)completion {
   }];
 }
 
+- (void)reCAPTCHAURLWithCompletion:(FIRReCAPTCHAURLCallBack)completion {
+  [self fetchAuthDomainWithCompletion:^(NSString *_Nullable authDomain,
+                                        NSError *_Nullable error) {
+    if (error) {
+      completion(nil, error);
+      return;
+    }
+    NSString *bundleID = [NSBundle mainBundle].bundleIdentifier;
+    NSString *clienID = _auth.app.options.clientID;
+    NSString *apiKey = _auth.requestConfiguration.APIKey;
+    NSString *languageCode = _auth.requestConfiguration.languageCode ?: @"en";
```

Comment:
Do not default to "en", which would override browser's default.

Reply:
Done.